src/Orleans/Orleans.csproj

```diff
@@ -74,8 +74,9 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="Microsoft.CSharp" />
-    <Reference Include="Newtonsoft.Json">
-      <HintPath>$(SolutionDir)packages\Newtonsoft.Json.6.0.8\lib\net45\Newtonsoft.Json.dll</HintPath>
+    <Reference Include="Newtonsoft.Json, Version=6.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
```

Comment:
Hah ;-) that's nuget, not me. But since in Orleans we're using everything from nuget packages I don't think that is hurts anyone feelings. This behavior I think is by default with nuget, that it is adding the full assembly name to references with version, etc...